[Master] Add guide on configuring login permissions for a user (#3486)
wso2/docs-is: en/docs/guides/identity-lifecycles/role-based-permissions.md

- **Super tenant level:** A role with super tenant permissions is used for managing all the tenants in the system and also for managing the key features in the system which are applicable to all the tenants.

- **Tenant level:** A role with tenant-level permissions is only applicable to individual tenant spaces.

You need to first have roles created in your tenant with the required permissions. You can then assign the role to users.

Users with the `admin` role are granted all permissions by default. Let's see how to configure a user with `admin` permissions.

## Configure admin permissions for a user

!!! Note "Prerequisites"
    You need to [create a new user]({{base_path}}/guides/identity-lifecycles/admin-creation-workflow/) on the management console.

To configure admin permissions for a user:

1. On the Management Console, go to **Users and Roles > List > Users**.
2. Click **Assign Roles** to assign all administrative permissions for the created user.
    ![Assign roles to Alex]({{base_path}}/assets/img/guides/assign-roles.png)
3. Select **admin** and click **Update**.
    ![Assign login permissions to Alex]({{base_path}}/assets/img/guides/assign-login-permissions.png)

> Your user will be granted login permissions for the Management Console.

Reviewer: ```suggestion
The user will be granted login permissions for the Management Console along with other admin permissions. See the complete list of [access permissions](#access-permissions) given below.
```

> The user will be granted login permissions for the Management Console along with other admin permissions. See the complete list of [access permissions](#access-permissions) given below.

Reviewer: ```suggestion
The user will be granted login permissions for the management console along with other admin permissions. See the complete list of [access permissions](#access-permissions) given below.
```

Permissions can be granted to a role at two levels:

- **Super tenant level:** A role with super tenant permissions is used for managing all the tenants in the system and also for managing the key features in the system, which are applicable to all the tenants.

> - **Tenant level:** A role with tenant level permissions is only applicable to individual tenant spaces.

Reviewer: ```suggestion
- **Tenant level:** A role with tenant-level permissions is only applicable to individual tenant spaces.
```

## Access permissions

> The permissions navigator that you use to enable permissions for a role is divided into these two categories (**Super Admin** permissions and **Admin** permissions) as shown below. However, note that there may be other categories of permissions enabled for a WSO2 product, depending on the type of features that are installed in the product.

Reviewer: ```suggestion
The permissions navigator that you use to enable permissions for a role is divided into two categories (**Super Admin** permissions and **Admin** permissions) as shown below. However, note that there may be other categories of permissions enabled for a WSO2 product, depending on the type of features that are installed in the product.
```

![]({{base_path}}/assets/img/fragments/role-permissions.png)

You can access the permissions navigator for a particular role by clicking **Permissions** as shown below.  

![]({{base_path}}/assets/img/fragments/permission-navigator.png)

By default, every WSO2 Identity Server comes with the default [user]({{base_path}}/references/concepts/user-management/users/) and
[role]({{base_path}}/references/concepts/user-management/roles-and-permissions/).

> - The **Admin** role has all the permissions in the system enabled by default. Therefore, this is a super tenant, with all permissions enabled.

Reviewer: ```suggestion
- The **Admin** role has all the permissions in the system enabled by default. Therefore, this is a super tenant with all permissions enabled.
```

-->

## Configure login permissions for a user

!!! Note "Prerequisites"
    You need to [create a new user]({{base_path}}/guides/identity-lifecycles/admin-creation-workflow/) on the management console.

> To configure login permissions to a user:

Reviewer: ```suggestion
To configure login permissions for a user:
```

1. On the Management Console, go to **Users and Roles > List > Users**.

> 2. Click on **Assign Roles** to assign login permissions for the created user.

Reviewer: ```suggestion
2. Click **Assign Roles** to assign login permissions for the created user.
```
